src/inmanta/moduletool.py

```diff
@@ -285,11 +299,96 @@ def init(self, output_dir: str, name: str, default: bool) -> None:
 
     def install(self) -> None:
         """
-        Install all modules the project requires.
+        install all modules the project requires.
         """
-        project: Project = self.get_project(load=False)
+        project: project = self.get_project(load=False)
         project.install_modules()
 
+    def update(self, module: Optional[str] = None, project: Optional[Project] = None) -> None:
```

Comment:
This method has moved from the `ModuleTool` to here. The only line that has changed in the implementation is the last line of `do_update`.